src/templates/static.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { graphql } from "gatsby"
+
+import SEO from "../components/seo"
```

Comment:
Wrong case in this component reference. Should be:
`import SEO from "../components/SEO"`

Reply:
fixed